iceoryx_hoofs/test/moduletests/test_posix_mutex.cpp

```diff
@@ -46,40 +46,62 @@ class Mutex_test : public Test
         }
     }
 
-    iox::posix::mutex sutNonRecursive{false};
-    iox::posix::mutex sutRecursive{true};
+    void signalThreadReady()
+    {
+        m_waitForThread.store(false);
+    }
+
+    void waitForThread()
+    {
+        while (m_waitForThread.load())
+        {
+        }
+    }
+
+    template <typename T>
+    int64_t getDuration(const T& start, const T& end)
+    {
+        return std::chrono::duration_cast<std::chrono::nanoseconds>(end - start).count();
+    }
+
+    static constexpr int64_t WAIT_IN_MS = 100;
+    static constexpr int64_t WAIT_IN_NS = WAIT_IN_MS * 1000000;
+
+    std::atomic_bool m_waitForThread{true};
+    iox::posix::mutex m_sutNonRecursive{false};
+    iox::posix::mutex m_sutRecursive{true};
     iox::units::Duration m_watchdogTimeout = 5_s;
```

Comment:
Why the `m_` prefix? This are all public members which should not have the prefix

Reply:
You are right but in my clang-tidy prints here a warning and then I let this thing fix it by clang-tidy since it is somehow our coding guideline.
I need to look for some clang-tidy rule where this rule only applies to non public members.

Reply:
we already use this style for publisher/subscriber options. Hopefully it's not to hard to find a solution